src/components/nabla/Pools/Swap/columns.tsx

```diff
@@ -49,14 +49,14 @@ export const reservesColumn: ColumnDef<SwapPoolColumn> = {
 } as const;
 
 export const aprColumn: ColumnDef<SwapPoolColumn> = {
-  header: 'APR',
+  header: 'APR(7d)',
```

Comment:
Let's add a space character. 
```suggestion
  header: 'APR (7d)',
```